apps/src/applab/designMode.js

```diff
@@ -159,8 +159,15 @@ function appendPx(input) {
   if (/px/.test(input)) {
     return input;
   }
-  return input ? input + 'px' : '';
+  //if any sort of object, including arrays, is inputted, return empty string.
+  //This gets around how parseInt and parseFloat treat arrays with first element numbers
+  if (typeof input === 'object'){
```

Comment:
@islemaster I'm interested in your opinion on this solution vs checking:
`(!(typeof input === 'number' || typeof input === 'string'))`